Model/Computed/WebpackChunks.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Magento\UpwardConnector\Model\Computed;
+
+use Magento\Framework\Filesystem\DirectoryList;
+use Magento\Framework\Filesystem\Driver\File;
+use Magento\Upward\DefinitionIterator;
+use Magento\UpwardConnector\Model\PageType;
+
+class WebpackChunks implements ComputedInterface
+{
+    public const SEARCH_PATTERN = 'RootCmp_{{SEARCH}}__';
+
+    /** @var \Magento\UpwardConnector\Model\PageType */
+    private $pageTypeResolver;
+
+    /** @var \Magento\Framework\Filesystem\DirectoryList */
+    private $directoryList;
+
+    /** @var \Magento\Framework\Filesystem\Driver\File */
+    private $driverFile;
+
+    public function __construct(
+        PageType $pageTypeResolver,
+        DirectoryList $directoryList,
+        File $driverFile
+    ) {
+        $this->pageTypeResolver = $pageTypeResolver;
+        $this->directoryList = $directoryList;
```

Comment:
Property is used only in constructor, perhaps we are dealing with dead code here.